### ISSUE TITLE
Missing methods `load()` and `dump()` in JSON

### ISSUE THREAD

**Issue by qsantos** [brython-dev/brython#466]
I found that the JSON module was implemented natively in Javascript in `libs/json.js`:

```
var $module = (function($B){

return  {
    loads : function(json_obj)
        return $B.jsobject2pyobject(JSON.parse(json_obj))
    },
    dumps : function(obj){return JSON.stringify($B.pyobject2jsobject(obj))},
}

})(__BRYTHON__)
```

However, only `loads()` and `dumps()` are implemented. I was unable to find how to write even a simple version of `load()`. I wanted to run `loads(fp.read())`, which is basically how it's done in CPython; can we do taht in Brython?

**Comment by PierreQuentel** [comment]
Yes, you can use `loads(fp.read())` ; I keep the issue open until I add `load()` and `dump()`.
Thanks !

**Comment by qsantos** [comment]
Oh, right, I meant that I wanted to implement load() in json.js by translating `loads(fp.read())` into Javascript. Changing the Python code does work (except for #465), but I am trying to change it as little as possible.
